subprojects/logging/src/main/java/org/gradle/internal/logging/services/DefaultLoggingManager.java

```diff
@@ -199,6 +205,7 @@ public void attachSystemOutAndErr() {
     private static class StartableLoggingRouter implements Stoppable {
         private final LoggingRouter loggingRouter;
         private LogLevel level;
+        private Integer maxWorkerCount;
```

Comment:
Also consider renaming this and all related methods as discussed in an earlier comment. Again, we may want to avoid the notion of "workers" as this doesn't relate to the actual workers infrastructure.